app/views/validators/_table.html.erb

```diff
@@ -342,23 +343,19 @@
       <td class="column-chart d-none d-lg-table-cell" id="skipped-after-<%=i%>">
         <%
           vl = chart_x_scale(validator.score&.skipped_after_history.to_a.count)
-          line_color = chart_line_color(validator.skipped_after_score)
-          fill_color = chart_fill_color(validator.skipped_after_score)
-          vector = validator.score&.skipped_after_history.to_a[-vl..-1].map { |v| (v.to_f*100.0).round(1) }
+          ma = chart_x_scale(validator.score&.skipped_after_moving_average_history.to_a.count)
+
+          skipped_slots_moving_avg = validator.score&.skipped_after_moving_average_history.to_a[-ma..-1]&.map { |v| (v.to_f*100.0).round(1) }
+          slot_vector = validator.score&.skipped_after_history.to_a[-vl..-1].map { |v| (v.to_f*100.0).round(1) }
+
+          line_color = chart_line_color(validator.skipped_slot_score)
         %>
-        <% max_value = vector&.max %>
-        <% if max_value && max_value > Y_SKIPPED_AFTER_MAX %>
-          <div class="chart-top-container">
-            <div class="chart-top-value" style="left: <%= max_value_position(vector) %>">
-              <%= max_value %>
-            </div>
-          </div>
-        <% end %>
-        <%= render 'validators/validator_small_chart',
+        <%= render 'validators/validator_skipped_slots_chart',
```

Comment:
I'd call this chart differently, since it's shared now, sth like validator_double_chart